locales/en/about-us.json

```
@@ -1 +1,5 @@
-{}
+{
+  "aboutUsSectionTitleIntro": "WE ARE COBDA,",
+  "aboutUsSectionTitleDescription": "THE PLACE WHERE BUYER AND SELLER MEETS.",
+  "aboutUsSectionDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc quis risus mi. Ut placerat qua lectus. Curabitur dictum velit non lacus ornare tempor. Nullam quis augue in leo aliquet malesuada sit amet eget eros. Sed laoreet posuere velit sit amet varius. Nulla eget purus non erat fringilla sodales vel a nulla. Cras sit amet tempus risus. Fusce dignissim blandit justo, eget elementum risus tristique a. Nunc fringilla lacus lacus, sit amet accumsan est pulvinar non. Praesent tristique enim lorem."
```

Since we already know it's from the `about-us` file, How about naming from its section instead? 

```suggestion
  "introductionSectionTitle": "WE ARE COBDA,",
  "introductionSectionSubtitle": "THE PLACE WHERE BUYER AND SELLER MEETS.",
  "introductionSectionDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc quis risus mi. Ut placerat qua lectus. Curabitur dictum velit non lacus ornare tempor. Nullam quis augue in leo aliquet malesuada sit amet eget eros. Sed laoreet posuere velit sit amet varius. Nulla eget purus non erat fringilla sodales vel a nulla. Cras sit amet tempus risus. Fusce dignissim blandit justo, eget elementum risus tristique a. Nunc fringilla lacus lacus, sit amet accumsan est pulvinar non. Praesent tristique enim lorem."
```

---

For key, use this:
```suggestion
  "aboutUsSectionTitle": "<0>WE ARE COBDA, <1></1> THE PLACE WHERE BUYER AND SELLER MEETS.</0>",
```
No more double title.
**REF** => https://www.npmjs.com/package/next-translate#6-use-html-inside-the-translation

---

If we use HTML inside so the render would be 
`<p>This is an example <b class="red">using HTML</b> inside the translation</p>`
but we have 2 separate `<h2>`. So, you'd suggest to wrap those `<h2>` together?